package.json

```
@@ -39,7 +39,7 @@
   },
   "devDependencies": {
     "eslint": "^8.1.0",
-    "typescript": "^4.8.0"
+    "typescript": "^4.8.2"
```

Can we change version range to be patch version or even fixed version? TS does not follow SemVer, so going from 4.8 to 4.9 may include unexpected breaking changes.
```suggestion
    "typescript": "~4.8.2"
```

---

This is a dev dependency, so I wouldn't bother. It's locked